### PR TITLE
Added time period for report & add fake data to the database

### PR DESCRIPTION
**Pull request by zaher-aa**

### CODE REVIEW

**Review by Ibrahim-Jarada — APPROVED** [review]

File: src/database/fakeData/fakeData.ts [hunk 1]
```diff
@@ -2,80 +2,66 @@ const settings = [
   {
     name: 'APP_MAIL',
     value: 'hello.world@gmail.com',
-    created_at: new Date(),
-    updated_at: new Date(),
+    // created_at: new Date(),
```
Comment (AymanSami98) [inline src/database/fakeData/fakeData.ts:5]: Remove unwanted comments

File: src/bin/index.ts [hunk 2]
```diff
@@ -114,9 +120,10 @@ const nextUpToOwedSplitPercentage = 0.5; // joi >=0 <=1
           association: ReportContentReportAssociation,
           include: [ContentReportWatchesAssociation],
         }],
+        transaction,
       },
     );
-
+    await transaction.commit();
     console.log('Success');
```
Comment (AymanSami98) [inline src/bin/index.ts:127]: Do you need this console?

**Review by AymanSami98 — APPROVED** [review]
Great job

File: src/bin/index.ts (hunk 2 above)
Comment (zaher-aa) [reply src/bin/index.ts:127]: Yes, to know that the process has finished successfully

**Review by RandSohail — APPROVED** [review]

File: src/database/fakeData/fakeData.ts (hunk 1 above)
Comment (zaher-aa) [reply src/database/fakeData/fakeData.ts:5]: We plan to use it later when creating migration seeds.

File: src/database/fakeData/fakeData.ts [hunk 3]
```diff
@@ -2,80 +2,66 @@ const settings = [
...
   },
 ];
 
 const roles = [
   {
     name: 'admin',
-    created_at: new Date(),
-    updated_at: new Date(),
+    // created_at: new Date(),
+    // updated_at: new Date(),
```
Comment (Jood80) [inline src/database/fakeData/fakeData.ts:44]: As long as those are commented you may remove them, and add them when it's needed.

**Review by Jood80 — CHANGES_REQUESTED** [review]

File: src/bin/index.ts (hunk 2 above)
Comment (Jood80) [reply src/bin/index.ts:127]: `console.info('some descriptive message')` ?

File: src/database/fakeData/fakeData.ts (hunk 3 above)
Comment (zaher-aa) [reply src/database/fakeData/fakeData.ts]: Fixed it

**Review by Jood80 — APPROVED** [review]

**Pull request merged**